packages/workers-playground/package.json

```diff
@@ -32,35 +32,26 @@
 		"@cloudflare/style-provider": "^3.1.0",
 		"@cloudflare/util-en-garde": "^8.0.10",
 		"@cloudflare/util-hooks": "^1.2.0",
+		"@cloudflare/workers-editor-shared": "workspace:*",
 		"@cloudflare/workers-tsconfig": "workspace:*",
-		"csstype": "^3.1.2",
-		"es-module-lexer": "^1.3.0",
 		"glob-to-regexp": "^0.4.1",
-		"hono": "^3.12.11",
 		"lz-string": "^1.5.0",
-		"react": "^18.2.0",
-		"react-dom": "^18.2.0",
-		"react-router-dom": "^6.15.0",
-		"react-split-pane": "^0.1.92",
-		"react-spring": "^8.0.27",
-		"react-tabs": "^6.0.2",
-		"react-use-websocket": "^3.0.0",
+		"react": "^17.0.2",
+		"react-dom": "^17.0.2",
```

Comment:
Why are we downgrading to React 17?

Reply:
Because the dashboard is at React 17, and since we're now publishing a shared component library to be used by both, it's easier to use the same react version for both clients. In particular https://github.com/facebook/create-react-app/issues/11769 makes it hard to publish a library for React 17 & 18 because the module reference for `react/jsx-runtime` needs to be `react/jsx-runtime.js` in React 17, but `react/jsx-runtime` in React 18. We're also not really using any React 18 features, and since we control both consumers of the `@cloudflare/workers-editor-shared` package it seemed to make sense to make things simpler for ourselves by only targeting one version of React.